src/catalog/column_catalog.cpp

```diff
@@ -142,9 +145,9 @@ bool ColumnCatalog::InsertColumn(oid_t table_oid,
                                  const std::vector<Constraint> &constraints,
                                  type::AbstractPool *pool,
                                  concurrency::TransactionContext *txn) {
+  (void) pool;
```

Comment:
you could probably use the  "UNUSED ATTRIBUTE" specifier instead.

Reply:
I am not sure why we would want to keep the parameter if we are not using it at all.